deploy/datadog/helm-values.yaml.example

```diff
@@ -0,0 +1,41 @@
+# This is a reasonable set of default HELM settings.
```

Comment:
I would add a comment with the chart version this was tested with (as it may break in the future)